src/core/pointcloud/qgspointcloudrenderer.h

```diff
@@ -189,6 +190,63 @@ class CORE_EXPORT QgsPointCloudRenderContext
           return;
       }
     }
+
+    //! Returns a map containing the attributes of a certain point from the point cloud
+    QMap<QString, QVariant> attributeMap( const char *data, std::size_t recordOffset, const QgsPointCloudAttributeCollection &attributeCollection )
```

Comment:
Same here -- this doesn't seem to be used anywhere anymore. Can it be removed now?

Reply:
It is used here https://github.com/qgis/QGIS/blob/19afdbe2359159da871f7d22b252c766095a87d9/src/core/pointcloud/qgspointclouddataprovider.cpp#L236

Reply:
Nevermind, I moved it to the data provider class